src/lib/bmx7_json/README.md

```diff
@@ -188,19 +181,21 @@ mlc115:~# cat /var/run/bmx6/json/options
                { "name": "json_originators", "help": "show originators in json format\n" }
              ]
 }
+```
 
-
-mlc115:~# cat /var/run/bmx6/json/parameters
+```
+mlc115:~# cat /var/run/bmx7/json/parameters
 { "OPTIONS": [
-               { "name": "plugin", "INSTANCES": [ { "value": "bmx6_json.so" } ] },
+               { "name": "plugin", "INSTANCES": [ { "value": "bmx7_json.so" } ] },
```

Comment:
is it possible you "beautify" this json?

Reply:
Yup

Reply:
Done